www/index.html

```diff
@@ -12,13 +12,12 @@
     <link href="//code.ionicframework.com/1.0.0-beta.14/css/ionic.css" rel="stylesheet">
     
     <script src="lib/ionic/js/ionic.bundle.js"></script>
+    <script src="http://code.ionicframework.com/collide/0.0.4/collide.js"></script>
```

Comment:
it may make sense to have this file local... external dependencies could make your application vulnerable to third party modifications